lua/lspconfig/util.lua

```diff
@@ -223,7 +223,7 @@ function M.get_managed_clients()
   return clients
 end
 
-function M._available_servers()
+function M.available_servers()
```

Comment:
```suggestion
--- @deprecated use `vim.lsp.confg` in Nvim 0.11+ instead.
function M.available_servers()
```

future: we should probably add `use vim.lsp.confg in Nvim 0.11+` to all util.lua deprecation notices

Reply:
Wait, people using nvim 0.10 won't have a way to remove this warning. I guess it's an acceptable tradeoff?

Reply:
Our nvim-lspconfig deprecation plan didn't really consider concerns about non-breaking soft *warnings* for Nvim 0.10, AFAIR. It doesn't strike me as too important, as long as their code doesn't break.